scripts/node.sh

```diff
@@ -72,7 +72,7 @@ parse_params() {
   # default values of variables set from params
   axelar_core_version=""
   reset_chain=0
-  root_directory="$HOME/.axelar_testnet"
+  root_directory=''
```

Comment:
Since default network is testnet, keep the default directory to testnet. Alternatively, we can remove the default network?

Reply:
For `root_directory` I just followed the pattern of `chain_id`: it starts empty and we set it according to testnet/mainnet.  I think it's better to do it that way because then we don't set these values in 2 places.

Reply:
Ah, yes, forgot about that, thanks